src/app/app.component.css

```diff
@@ -0,0 +1,3 @@
+.custom-margin-top {
```

Comment:
Can use existing bootstrap  "mt" class for margin top no need to create a separate CSS class

Reply:
The bootstrap class "mt" only adds margin till mt-5. More margin was required, therefore created a separate class.

Reply:
![image](https://github.com/Kanika-Mahajan27/angular-login-app/assets/72973397/f419f354-14ca-44bd-ae33-053cb187f679)
This layout causes issues at smaller screen widths if you absolutely have to use extra classes try to use relative units instead of fixed units like px.